src/libraries/Native/Unix/System.Globalization.Native/pal_icushim.c

```diff
@@ -6,6 +6,7 @@
 #if defined(TARGET_UNIX)
 #include <dlfcn.h>
 #elif defined(TARGET_WINDOWS)
+#include <icu.h>
```

Comment:
`pal_icushim_internal.h` imports `icu.h`. Maybe just move that import to the top of this file?

https://github.com/dotnet/runtime/blob/51fdfe19ce743c16274ebaa36f9a92b746701621/src/libraries/Native/Unix/System.Globalization.Native/pal_icushim_internal.h#L46